wire-runtime/build.gradle

```diff
@@ -12,6 +12,7 @@ kotlin {
         metaInfo = true
       }
     }
+    nodejs()
```

Comment:
curious: what does this do?

Reply:
If I understand it correctly now it's mandatory to declare a real JS target for the tests to run.

Reply:
We probably also want `browser()`

Reply:
Good call.